poster/poster_file.tex

```diff
@@ -146,10 +156,14 @@ \section*{Improving GrammarGuru suggestions}
 \begin{center}
 \includegraphics[width=0.8\linewidth]{proj_img}
```

Comment:
Was there supposed to be an image here? I didn't see it in the final PDF.

Reply:
It’s here https://github.com/luizperes/training-grammar-guru/blob/cmpt419/poster/proj_img.png